src/test/java/javaslang/control/OptionTest.java

```diff
@@ -8,10 +8,7 @@
 import javaslang.Serializables;
 import org.junit.Test;
 
-import java.util.Iterator;
-import java.util.NoSuchElementException;
-import java.util.Objects;
-import java.util.Optional;
+import java.util.*;
 
```

Comment:
I usually set idea to never use wildcard import such as `import java.util.*;` Now i use setting from javaslang standard.